### PR TITLE
tests_l2: add L1 and L2 test for media feature validation

### PR DESCRIPTION
**Pull request by mregmi**
For L1 test we are using vainfo and for L2 we are using libvpl tests.

### CODE REVIEW

**Review by vbedida79 — APPROVED** [review]

File: tests/l2/dgpu/intelvpl_build.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+              libva libva-devel libdrm libdrm-devel
+        RUN dnf install -y 'dnf-command(config-manager)' && \
+            dnf config-manager --add-repo \
+            https://repositories.intel.com/gpu/rhel/9.2/lts/2350/unified/intel-gpu-9.2.repo
+        RUN dnf -y update && \
+            dnf install -y libva-utils intel-gmmlib-devel libvpl2 libvpl-devel libvpl-tools \
+              libmfx
+        RUN git clone -b intel-media-23.4.3 --single-branch https://github.com/intel/media-driver.git && \
+              cd media-driver && mkdir media-driver build && cd build && \
+                cmake -D ENABLE_PRODUCTION_KMD=ON ../ && make -j 8 && make install
```
Comment (uMartinXu) [inline tests/l2/dgpu/intelvpl_build.yaml:40]: Please try to use
make -j `nproc`
to replace 
make -j 8

File: tests/l2/dgpu/intelvpl_build.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        # we need this ony for test samples
```
Comment (uMartinXu) [inline tests/l2/dgpu/intelvpl_build.yaml:41]: we need this for testing samples

File: tests/l2/dgpu/vainfo_build.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            dnf install -y flex bison gcc gcc-c++ make autoconf libtool cmake git gdb \
+              libva libva-devel libdrm-devel
+        RUN dnf install -y 'dnf-command(config-manager)' && \
+            dnf config-manager --add-repo \
+            https://repositories.intel.com/gpu/rhel/9.2/lts/2350/unified/intel-gpu-9.2.repo
+        RUN dnf -y update && \
+            dnf install -y libva-utils intel-gmmlib-devel
+        RUN git clone -b intel-media-23.4.3 --single-branch https://github.com/intel/media-driver.git && \
+              cd media-driver && mkdir media-driver build && cd build && \
+                cmake -D ENABLE_PRODUCTION_KMD=ON ../ && make -j 8 && make install
```
Comment (uMartinXu) [inline tests/l2/dgpu/vainfo_build.yaml:39]: see comments above.

File: tests/l2/dgpu/intelvpl_build.yaml (hunk 2 above)
Comment (mregmi) [reply tests/l2/dgpu/intelvpl_build.yaml]: Fixed

File: tests/l2/dgpu/vainfo_build.yaml (hunk 3 above)
Comment (mregmi) [reply tests/l2/dgpu/vainfo_build.yaml]: Done

**Review by uMartinXu — APPROVED** [review]

**Pull request merged**